samples/apps/copilot-chat-app/scripts/2_init-backend.cmd

```diff
@@ -0,0 +1,15 @@
+@echo off
```

Comment:
Both of the "init" scripts actually run the software. Consider removing the numbers and renaming:
- "install-preprequisites.cmd"
- "start-backend.cmd"
- "start-frontend.cmd"
- "start.cmd"